WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -10,6 +10,11 @@ import Alamofire
     case socialLoginExistingUserUnconnected
     case invalidTwoStepCode
     case unknownUser
+    /// The API response requires handling MFA, but the caller didn't specify a handler.
+    ///
+    /// Use to ensure backwards compatibility in `authenticateWithUsername(...)` calls after support for the MFA closure
+    /// was introduced.
+    case noMultifactorHandlerGiven
```

Comment:
Strictly speaking, adding a `case` to an `enum` could be considered a breaking change though, because if the client code already has a `switch` on that error value, the switch which was exhaustive before will become non-exhaustive (except if it has an `@unknown default` case)

Reply:
Thanks for catching this @AliSoftware... How embarrassing: Tries to fix a breaking change, introduces a breaking change. I have a feeling there's a meme for that.

It turns out that I was too optimistic in my approach anyway. The fix addresses the API compatibility in Swift, but not Objective-C, and that's where the issue actually is. It was written in plain sight [in the CI error](https://buildkite.com/automattic/woocommerce-ios/builds/16542#018b2c09-3960-48f0-93fb-b430349c1dd2/759-2868), if only I had looked properly:

<img width="1177" alt="image" src="https://github.com/wordpress-mobile/WordPressKit-iOS/assets/1218433/db1ee5a2-46ab-4c46-b199-d00953809031">

It's already late in my day... I'm going to start the code freeze without updating WordPressKit and continue to work on this tomorrow 😞